src/state/nftMarket/reducer.ts

```diff
@@ -1,9 +1,9 @@
 import { createSlice, createAsyncThunk } from '@reduxjs/toolkit'
-import { getCollectionsApi, getCollectionsSg, getNftsMetadata, getNftsMarketData } from './helpers'
-import { State, Collection, NFT } from './types'
+import { getNftsMetadata, getNftsMarketData, getCollectionsApi, getCollectionsSg } from './helpers'
+import { State, Collection, NFT, NFTMarketLoadingState } from './types'
 
 const initialState: State = {
-  isInitializing: true,
+  loadingState: NFTMarketLoadingState.IDLE,
```

Comment:
This pattern is more flexible and is used in other states.

Reply:
We probably want to make a difference between the first loading and other loadings